src/components/TaskEntry.js

```diff
@@ -0,0 +1,49 @@
+import styled from 'styled-components/macro'
+
+export default function TaskEntry({ onClick, date, nameOfCrop, tasks }) {
+  return (
+    <Wrapper>
+      <input type="checkbox" onClick={onClick} />
+      <Grid>
+        <h2>{formatDate(date)}</h2>
+        {nameOfCrop}: {tasks}
+      </Grid>
+    </Wrapper>
+  )
+  function formatDate(date) {
+    const options = {
+      weekday: 'long',
+      month: 'long',
+      day: 'numeric',
+    }
+    console.log(date)
+    const newDate = new Date(date)
+    const formatedDate = newDate.toLocaleDateString('en-US', options)
```

Comment:
I think English is the default but I don't know if you can leave it. anyway probably it is more clearly like you did it.